portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,49 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Mariah Crawford's Portfolio</h1>
+      <p>A Princeton University student pursuing a Bachelor of Science in Engineering (BSE) 
+          in Computer Science and certificates in Entrepreneurship and Theater. Passionate 
+          about using her creativity to explore the intersections of technology and entrepreneurship. 
+          Exhibits strong computer programming skills in Java and Python languages and is a highly 
+          motivated individual and a quick learner. Demonstrates great leadership skills through her 
+          involvement as an officer in the diSiac Dance Company, Student Government (USG), and the 
+          Entrepreneurship Club.</p>
+      <img src="images/me.jpg" style="width:700px;height:500px;"/>
+
+      <p>Click here to get a random fun fact about Mariah:</p>
+      <button onclick="addRandomGreeting()">Generate Fun Fact</button>
+      <div id="funfact-container"></div>
```

Comment:
Nit: Rename this to fun-fact-container since we are separating different words using hyphens.